tests/file_loading.py

```diff
@@ -0,0 +1,44 @@
+"""
+File:           file_loading.py
+Description:    Constants and utilities used for loading files/recipes.
+"""
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Final
+
+from conda_recipe_manager.parser.recipe_parser import RecipeParser
+from conda_recipe_manager.parser.recipe_parser_convert import RecipeParserConvert
+
+# Path to supplementary files used in test cases
+TEST_FILES_PATH: Final[str] = "tests/test_aux_files"
```

Comment:
this bit here will work only if you execute the tests from the root repository.
I would recommend getting it relative of the test file maybe? or using ``pytest-datadir``
something like

```python
TEST_FILES_PATH = Path(__file__).parent / "test_aux_files"
```

Reply:
Fair point. I almost always run the tests from the `make` directives so I haven't personally had that issue.